### DISCUSSION TITLE
`commitMessageExtra` customization

### DISCUSSION THREAD

**Discussion by vicasas** [renovatebot/renovate#30068] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

For the following `package.json`, I want to get an output for the PR message of the form:

```text
chore(deps): Bump @types/node from ^20.14.7 to ^20.14.10
```

```json
{
    ...
    "devDependencies": {
        "@types/node": "^20.14.7"
    }
}
```

Previously, to achieve the above, it was enough to add the following configuration:

```json
{
    ...
    "commitMessageExtra": "from {{currentValue}} to {{newValue}}",
    ...
}
```

Which had the following output:

```text
chore(deps): Bump @types/node from ^20.14.7 to ^20.14.10
```

But now when using `newValue` it seems to work differently, as I get the following output:

```text
chore(deps): Bump @types/node from ^20.14.7 to ^20.14.7
```

The `newVersion` is equal to the "currentVersion", the strange thing is that this does not happen in all packages, only in some.

<details>
<summary>Bump @&#8203;types/node to ^20.14.7</summary>

  - Schedule: ["on sunday before 6:00am"]
  - Branch name: `renovate/node-20.x-lockfile`
  - Merge into: `main`
  - Upgrade [@types/node](https://togithub.com/DefinitelyTyped/DefinitelyTyped) to `20.14.10`

</details>

So how can I modify the Renovate configuration to achieve the necessary formatting? More than anything I'm interested in being able to add `from <version>`.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
Can you attach the logs from the relevant run?
